aicssegmentation/structure_wrapper/WorkflowStep.py

```diff
@@ -0,0 +1,40 @@
+import importlib
+import numpy as np
+
+class WorkflowStep:
+    """
+    A class that defines a step in an AICS-Segmentation workflow.
+    """
+    def __init__(self, step_config: dict):
+        """
+        Constructor for the workflow object
+
+        Params:
+            step_config (dict): dictionary object containing information about this workflow step
+        """
+        self.name: str = step_config["name"]         # Name of the workflow step
+        self.parent: int = step_config["parent"] - 1 # Index of parent in entire workflow #TODO: Better to change json to 0-indexed to avoid confusion
+        self.result: np.ndarray = None               # Result of running this step, None if not executed
+
+        module = importlib.import_module(step_config["module"])
+        self.__function = getattr(module, step_config["function"])
+        self.__parameters: dict = None
+        if "parameter" in step_config:
+            self.__parameters = step_config["parameter"]
+
+    def execute(self, image: np.ndarray) -> np.ndarray:
```

Comment:
It would be better to use `Union[np.ndarray, List]`. Basically, the input can be a single image or a list of images. 

See https://github.com/AllenCell/aics-segmentation/blob/feature/json_workflow/aicssegmentation/structure_wrapper_config/structure_config_utils.py#L37

to test, you may use this example: https://github.com/AllenCell/aics-segmentation/blob/feature/json_workflow/aicssegmentation/structure_wrapper_config/conf_lmnb1_interphase.json

Reply:
Yes, it looks like we need to accept a List, or a variable number of positional parameters (ie. `*images`)

Reply:
When it is a list of images, do I have to execute the function on each image on the list or will it be fine to just pass it to your `aics-segmentation` functions?

Reply:
@jxchen01 Still curious about my above question, please let me know!